src/test/java/seedu/address/logic/commands/tagcommands/TagEditCommandTest.java

```diff
@@ -50,7 +50,7 @@ public void execute_noFieldSpecifiedUnfilteredList_success() {
         TagEditCommand tagEditCommand = new TagEditCommand(INDEX_FIRST_TAG, new EditTagDescriptor());
         Tag editedTag = model.getFilteredTagList().get(INDEX_FIRST_TAG.getZeroBased());
 
-        String expectedMessage = String.format(TagEditCommand.MESSAGE_EDIT_TAG_SUCCESS, editedTag);
+        String expectedMessage = String.format(TagEditCommand.MESSAGE_EDIT_TAG_SUCCESS, editedTag, editedTag);
```

Comment:
just checking, is this correct?

Reply:
yep cuz I realised Edited tag: [original tagname] does not make sense so I change to Edited tag: [original tagname] to [new tagname]. Then for this test, since no field is specified, the original and new tagname are the same

Reply:
ah okok thanks!